src/sentry/static/sentry/app/api.jsx

```diff
@@ -166,6 +166,7 @@ export class Client {
     let query;
     try {
       query = $.param(options.query || '', true);
```

Comment:
Should we not remove the old line here?

Reply:
He got it in a separate PR i think

Reply:
Yea I got it here https://github.com/getsentry/sentry/pull/14061

git stash failed me 😢